packages/botonic-core/src/hubtype-service.ts

```diff
@@ -30,18 +29,10 @@ interface HubtypeServiceArgs {
   server: ServerConfig
 }
 
-const _WEBCHAT_PUSHER_KEY_ = getWebpackEnvVar(
-  // @ts-ignore
-  typeof WEBCHAT_PUSHER_KEY !== 'undefined' && WEBCHAT_PUSHER_KEY,
-  'WEBCHAT_PUSHER_KEY',
-  '434ca667c8e6cb3f641c' // pragma: allowlist secret
-)
-const _HUBTYPE_API_URL_ = getWebpackEnvVar(
-  // @ts-ignore
-  typeof HUBTYPE_API_URL !== 'undefined' && HUBTYPE_API_URL,
-  'HUBTYPE_API_URL',
-  'https://api.hubtype.com'
-)
+const WEBCHAT_PUSHER_KEY =
+  process.env.WEBCHAT_PUSHER_KEY || '434ca667c8e6cb3f641c'
```

Comment:
Doubt: Should we still add the `// pragma: allowlist secret` comment with this pusher key to avoid security warnings?